core/cmd/wandb-core/main.go

```diff
@@ -42,15 +42,15 @@ func main() {
 	}
 
 	// set up sentry reporting
-	observability.InitSentry(*disableAnalytics, commit)
+	observability.InitSentry("", *disableAnalytics, commit)
```

Comment:
it would be better to pass the DSN explicitly here; otherwise it looks like we're disabling Sentry

since `sentry.go` is now used by both core and the Go client and the `defaultSentryDsn` is specific to core, the constant should be moved into this file and used here